deploy/examples/cluster.yaml

```diff
@@ -97,15 +97,15 @@ spec:
     # enable the Multus network provider
     #provider: multus
     #selectors:
-      # The selector keys are required to be `public` and `cluster`.
```

Comment:
?

Reply:
You can revert this file, right? The editor is changing the spacing automatically, we should probably fix this in a separate PR so everyone doesn't have this issue.

Reply:
reverted this commit.